docs/reference/eql/detect-threats-with-eql.asciidoc

```diff
@@ -0,0 +1,424 @@
+been normalized and mapped to the {ecs-ref}[Elastic Common Schema (ECS)]. The
+dataset includes events that imitate behaviors related to regsvr32 misuse, as
+documented by MITRE ATT&CK®.
+
+To get started, download and index the dataset:
+
+. Download the https://raw.githubusercontent.com/elastic/elasticsearch/{branch}/docs/src/test/resources/normalized-T1117-AtomicRed-regsvr32.json[`normalized-T1117-AtomicRed-regsvr32.json`] dataset.
+
+. Index the data into the `my-index-000001` index with the following
+<<docs-bulk,bulk API>> request:
++
+[source,sh]
+----
+curl -H "Content-Type: application/json" -XPOST "localhost:9200/my-index-000001/_bulk?pretty&refresh" --data-binary "@normalized-T1117-AtomicRed-regsvr32.json"
+----
+// NOTCONSOLE
+
+. Use the <<cat-indices,cat indices API>> to verify the dataset was indexed
+successfully.
++
+[source,console]
+----
+GET /_cat/indices/my-index-000001?v&h=health,status,index,docs.count
+----
+// TEST[setup:atomic_red_regsvr32]
++
+The API returns the following response. The `docs.count` value should show `150`
+documents were indexed.
++
+[source,txt]
+----
+health status index            docs.count
+yellow open   my-index-000001         150
+----
+// TESTRESPONSE[non_json]
+
+[discrete]
+[[eql-ex-get-a-count-of-regsvr32-events]]
+=== Get a count of `regsvr32.exe` events
+
+Since you're looking for `regsvr32` misuse, start by getting a count of events
+using `regsvr32.exe`.
+
+The following <<eql-search-api,EQL search API>> request uses an EQL query to
+retrieve a count of events with a `process.name` of `regsvr32.exe`. The query
+starts with the <<eql-syntax-match-any-event-category,`any where` keywords>>,
+meaning the query can match events of any <<eql-required-fields,event
+category>>.
+
+[source,console]
+----
+GET /my-index-000001/_eql/search?filter_path=-hits.events  <1>
+{
+  "query": """
+    any where process.name == "regsvr32.exe"        <2>
+  """,
+  "size": 150                                       <3>
+}
+----
+// TEST[setup:atomic_red_regsvr32]
+
+<1> Uses the `?filter_path=-hits.events` query parameter to exclude the
+`hits.events` property from the response. The `hits.events` property contains
+the document source for any matching events. This request is intended to
+retrieve a count of events only.
+<2> Uses an EQL query to match events with a `process.name` of `regsvr32.exe`.
+<3> Returns up to 150 events or sequences matching the EQL query.
+
+The request returns the following response, indicating that 143 events match the
+query.
+
+[source,console-result]
+----
+{
+  "is_partial": false,
+  "is_running": false,
+  "took": 60,
+  "timed_out": false,
+  "hits": {
+    "total": {
+      "value": 143,
+      "relation": "eq"
+    }
+  }
+}
+----
+// TESTRESPONSE[s/"took": 60/"took": $body.took/]
+
+[discrete]
+[[eql-ex-check-for-command-line-artifacts]]
+=== Check for command line artifacts
+
+Your previous query matched 143 events. Many of these events are likely to be
+false positives.
```

Comment:
I wouldn't say they are false positives, but more that they are superfluous. Like you know all the things that regsvr32.exe did -- it loaded some DLLs, dropped some files, made a network connection. But maybe it's better to show in this example that you want to know _how regsvr32.exe was called in the first place_. That's often the context you want to know when hunting -- where did this come from, and how?

I think by articulating that goal, it makes it more clear why you want to look for the process creation event: it contains `user.name`, which says what user executed the process, as well as `process.parent.name`, so you know where it came from, and `process.command_line`, which gives you information of everything that regsvr32.exe was  told to do. That's why we end up valuing the process creation events so much.

I think if you add those points, it'll add more punch/story and insight into the thought process to the example.